Source/bmalloc/bmalloc/TZoneHeap.h

```diff
@@ -220,7 +278,7 @@ private: \
 
 
 #define MAKE_BTZONE_MALLOCED_TEMPLATE_IMPL(_templateParameters, _type) \
-    _templateParameters ::bmalloc::api::HeapRef _type::s_heapRef
+    _templateParameters ::bmalloc::api::HeapRef _type::s_heapRef;
```

Comment:
Please remove this semicolon at the end.  Adding it here will result in an extra semicolon in all the places that uses it.  I wonder why your build isn't already failing.

Reply:
This wouldn't cause the build to fail -- an extra semicolon is essentially a no-op.

(Obviously I'll remove it, just justifying how I was able to build/run tests &c)